pkg/oc/cli/cmd/process.go

```diff
@@ -74,48 +76,79 @@ var (
 	cmd.MarkFlagFilename("param-file")
-	cmd.Flags().Bool("ignore-unknown-parameters", false, "If true, will not stop processing if a provided parameter does not exist in the template.")
-	cmd.Flags().BoolP("local", "", false, "If true process the template locally instead of contacting the server.")
-	cmd.Flags().BoolP("parameters", "", false, "If true, do not process but only print available parameters")
-	cmd.Flags().StringP("labels", "l", "", "Label to set in all resources for this template")
-
-	cmd.Flags().StringP("output", "o", "json", "Output format. One of: describe|json|yaml|name|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=...")
-	cmd.Flags().Bool("raw", false, "If true, output the processed template instead of the template's objects. Implied by -o describe")
-	cmd.Flags().String("output-version", "", "Output the formatted object with the given version (default api-version).")
-	cmd.Flags().StringP("template", "t", "", "Template string or path to template file to use when -o=go-template, -o=go-templatefile.  The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview]")
-
-	// kcmdutil.PrinterForCommand needs these flags, however they are useless
-	// here because oc process returns list of heterogeneous objects that is
-	// not suitable for formatting as a table.
-	cmd.Flags().BoolP("show-all", "a", false, "When printing, show all resources (default hide terminated pods.)")
-	cmd.Flags().Bool("show-labels", false, "When printing, show all labels as the last column (default hide labels column)")
-	cmd.Flags().Bool("no-headers", false, "When using the default output, don't print headers.")
-	cmd.Flags().MarkHidden("no-headers")
-	cmd.Flags().String("sort-by", "", "If non-empty, sort list types using this field specification.  The field specification is expressed as a JSONPath expression (e.g. 'ObjectMeta.Name'). The field in the API resource specified by this JSONPath expression must be an integer or a string.")
-	cmd.Flags().MarkHidden("sort-by")
-
+	cmd.Flags().BoolVar(&o.ignoreUnknownParams, "ignore-unknown-parameters", o.ignoreUnknownParams, "If true, will not stop processing if a provided parameter does not exist in the template.")
+	cmd.Flags().BoolVarP(&o.local, "local", "", o.local, "If true process the template locally instead of contacting the server.")
+	cmd.Flags().BoolVarP(&o.parameters, "parameters", "", o.parameters, "If true, do not process but only print available parameters")
+	cmd.Flags().StringVarP(&o.labels, "labels", "l", o.labels, "Label to set in all resources for this template")
+
+	cmd.Flags().BoolVar(&o.raw, "raw", o.raw, "If true, output the processed template instead of the template's objects. Implied by -o describe")
+
+	// REBASE-FIXME: we need to wire a custom printer for this command
+	// modify description on "output" flag
+	//outputFlag := cmd.Flags().Lookup("output")
+	//if outputFlag != nil {
```

Comment:
@deads2k The intent here was to have a custom printer for this command that could use the humanreadable printer in the case of `--output=describe`. It would be better to remove support for this kind of output from this command and stick with the normal genericclioptions printer